tools/gemma/run_gemma_xla.py

```diff
@@ -31,6 +46,47 @@
 from gemma.model_xla import GemmaForCausalLM
 from gemma.tokenizer import Tokenizer
 
+"""
+Sample usage:
+
+Run the verification script supplying your model size, converted checkpoint file,
+vocabulary file, and test prompt.
+
+```
+python keras-nlp-gemma/tools/gemma/run_gemma_xla.py \
+  --size 2b \
+  --checkpoint_file fine_tuned_imdb.ckpt \
+  --vocab_file gemma_tokenizer/vocabulary.spm \
+  --prompt "Three Billboards"
+```
+
+After several minutes, this should produce:
```

Comment:
is this for cpu only? maybe say, 

After a delay (a couple minutes if running on CPU), this should produce:

Reply:
Good idea, changed the wording to say that.